x/epochs/keeper/hooks.go

```diff
@@ -4,10 +4,12 @@ import (
 	sdk "github.com/cosmos/cosmos-sdk/types"
 )
 
+// AfterEpochEnd the first block whose timestamp is after the duration is counted as the end of the epoch
```

Comment:
it was a little unclear to me what the comments meant when I first read it. Consider a more specified comment!